### PR TITLE
844977: Provided the UG documentation for sort icon template and cell right tapped supports.

### PR DESCRIPTION
**Pull request by RasikaPalanisamy**
Provided the UG documentation for sort icon template and cell right tapped supports.

### CODE REVIEW

**Review by Chrispineagunja — COMMENTED** [review]
Content review completed.

File: MAUI/DataGrid/selection.md [hunk 1]
```diff
@@ -356,7 +356,8 @@ The `Datagrid` provides the following Events to handle interactions to the cells
 
 * [CellTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellTapped) : Called when a tap with a cell has occurred.
 * [CellDoubleTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellDoubleTapped) : Called when the user has tapped a cell with a primary button at the same cell twice in quick succession.
-* [CellLongPress](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellLongPress) : Called when a long-press gesture with a primary button has been recognized for a cell. 
+* [CellLongPress](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellLongPress) : Called when a long-press gesture with a primary button has been recognized for a cell.
+* `CellRightTapped` : Called when a right tap with a cell has occurred.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/selection.md]: Addressed the concerns provided.

File: MAUI/DataGrid/selection.md [hunk 2]
```diff
@@ -420,6 +421,29 @@ private void dataGrid_CellLongPress(object sender,DataGridCellLongPressEventArgs
 {% endhighlight %}
 {% endtabs %}
 
+### CellRightTapped event
+This event will be triggered when a cell is right-clicked using a mouse, right-clicked using a touchpad, or when a long-pressed touchscreen interaction is released. This event has the `DataGridCellRightTappedEventArgs` as arguments.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/selection.md]: Addressed the concerns provided.

**Review by ashok-kuvaraja — APPROVED** [review]
The code changes are fine.

**Pull request merged**